### PR TITLE
Update docs/make.jl for Documenter 0.24

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by dpo** [comment]
What is this for? There is no explanation.

**Comment by amontoison** [comment]
I'm working on an MathOptNLPModel that create an NLPModel from a JuMP model.
I interfaced MOI functions available for an MOI.AbstractNLPEvaluator with generic NLPModel functions.
https://github.com/JuliaOpt/MathOptInterface.jl/blob/master/src/nlp.jl

But `objective_expr` and `constraint_expr` are the only functions that are not available in NLPModel.
I think that it can be useful to get expression tree of the objective function / constraints in some cases.

**Comment by dpo** [comment]
It will be specific to MathOptNLPModels though, right? No other model can return an expression?!

**Comment by amontoison** [comment]
Yes, to my knowledge it's the only one that has this feature.

**Comment by dpo** [comment]
Is it useful to define those methods here then? Shouldn't they just be in NLPModelsJuMP?

**Comment by amontoison** [comment]
Yes, you're right.
I will just update Documenter with this pull request.

File: docs/make.jl [hunk 1]
```diff
@@ -19,8 +16,4 @@ makedocs(
           ]
 )
 
-deploydocs(deps = nothing, make = nothing,
```
Comment (amontoison) [inline docs/make.jl]: Other keywords used they value by default, so we can remove them.
I'm still trying to find why, after deployment, links to source don't work...

**Comment by dpo** [comment]
Thank you.

**Pull request merged**